app/http/HttpFilter.scala

```diff
@@ -52,3 +53,8 @@ final class HttpFilter(env: Env)(using val mat: Materializer)(using Executor)
     if HTTPRequest.isApiOrApp(req)
     then result.withHeaders(headersForApiOrApp(using req)*)
     else result.withHeaders(permissionsPolicyHeader)
+
+  private def addEmbedderPolicyHeaders(req: RequestHeader)(result: Result) =
+    if !embedderPolicy.isSet(result) && embedderPolicy.supportsCoepCredentialless(req)
```

Comment:
This isn't quite right, it's a bit confusing, while Firefox supports the credentialless header, it doesn't yet support credentialless iframes which means using the credentialless header on pages with iframes in Firefox will break them. Also, credentialless iframes are only supported since Chrome 113 while the header is supported on 96+.

So we need a different check `supportsCredentiallessIframes` here which currently is only Chrome 113. Looks like the UaMatcher `isChrome113Plus` even still exists.